src/platform/bouffalolab/BL602/CHIPPlatformConfig.h

```diff
@@ -56,19 +56,18 @@
 #define CHIP_CONFIG_HASH_IMPLEMENTATION_MBEDTLS 0
 #define CHIP_CONFIG_HASH_IMPLEMENTATION_PLATFORM 0
 
-// FIXME: BL602 set to MBED-TLS (But this is third-party repo in CHIP, not SDK)
-
 #define CHIP_CONFIG_AES_IMPLEMENTATION_OPENSSL 0
 #define CHIP_CONFIG_AES_IMPLEMENTATION_AESNI 0
 #define CHIP_CONFIG_AES_IMPLEMENTATION_MBEDTLS 1
 #define CHIP_CONFIG_AES_IMPLEMENTATION_PLATFORM 0
 
-// FIXME: BL602 currently set to CHIP (Does this use Entropy.cpp ?)
-
 #define CHIP_CONFIG_RNG_IMPLEMENTATION_OPENSSL 0
 #define CHIP_CONFIG_RNG_IMPLEMENTATION_CHIPDRBG 1
 #define CHIP_CONFIG_RNG_IMPLEMENTATION_PLATFORM 0
 
+//#define CHIP_CONFIG_SHA256_CONTEXT_SIZE sizeof(bl_sha_ctx_t)
+#define CHIP_CONFIG_SHA256_CONTEXT_SIZE ((1 + 5 + 18 + 16 + 16) * sizeof(unsigned int))
```

Comment:
Are you certain this is accurate?

Reply:
Yes.
With Bouffalolab BL602 crypto engine enabled, CHIP_CONFIG_SHA256_CONTEXT_SIZE should be sizeof(bl_sha_ctx_t). 
sizeof(bl_sha_ctx_t)  is ((1 + 5 + 18 + 16 + 16) * sizeof(unsigned int)).